AlicerceTests/Extensions/OptionalTests.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  OptionalTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+final class OptionalTests: XCTestCase {
+
+    func testThen_UsingNonOptionalValue_ShouldUnwrapTheValue() {
+        let anOptionalString: String? = "😎"
+
+        anOptionalString.then { value in
+            XCTAssert(true, "🚀 \(value) unwrapped ✅")
+        }
+    }
```

Comment:
Do you think there would be any merit in adding a test for when the string is nil?